applications/serial_lte_modem/Kconfig

```
@@ -81,8 +81,16 @@ config SLM_START_SLEEP
 	help
 	  Put nRF9160 into deep sleep mode after startup.
 
-config SLM_INTERFACE_PIN
-	int "Interface GPIO to wake up or exit idle mode"
+config SLM_WAKEUP_PIN
+	int "Interface GPIO to exit from sleep or idle mode"
+
+config SLM_INDICATE_PIN
+	default -1
+	int "Interface GPIO to indicate data available or unexpected reset"
+
+config SLM_INDICATE_TIME
+	default 100
+	int "Indicate GPIO active time in milliseconds"
```

it's a good custom to have the type (and title) of the setting at top.

Also, the title is a bit long, some of the title more belongs in the help text of the setting.

Proposal:
```suggestion
	int "SLM indication period"
	default 100
	help
	  GPIO active indication time in milliseconds. This setting specify the period length for the pin to be active.
```